src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -221,6 +221,7 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' != 'netcoreapp'">
     <Compile Include="System\StringSplitExtensions.cs" />
+    <Compile Include="System\Runtime\CompilerServices\RuntimeWrappedExceptionTests.cs" />
```

Comment:
This is in the wrong group.  It needs to be in https://github.com/zaytsev-victor/corefx/blob/88cc13ee3546d8656743ebe61790f9d5f8de7ce9/src/System.Runtime/tests/System.Runtime.Tests.csproj#L176.